### ISSUE TITLE
SA1002 fired for a semicolon that is the last character in a document

### ISSUE THREAD

**Issue by vweijsters** [DotNetAnalyzers/StyleCopAnalyzers#2699]
The following code:
```C#
using ExampleOfFalsePositive = System.ValueTuple<System.Collections.IList, int>;
```
raises SA1002 when the semicolon is the last character in the document (not followed by a newline). This should not raise any diagnostic.
